pkgs/development/mobile/flashtool/default.nix

```
@@ -12,11 +12,12 @@
 #   instructions fix the executable bits already.
 
 stdenv.mkDerivation rec {
-  name = "flashtool-0.9.14.0";
+  pname = "flashtool";
+  version = "0.9.14.0";
 
   src = requireFile {
     url = "http://dfiles.eu/files/n8c1c3pgc";
```

```suggestion
    url = "https://dfiles.eu/files/n8c1c3pgc";
```

~~That site looks very questionable. I would be for droping this package.~~

```
This file does not exist, the access to the following file is limited or it has been removed due to infringement of copyright. 
```

Lets remove it.